resources/machines/fdmprinter.json

```diff
@@ -17,6 +17,12 @@
             "description": "Gcode commands to be executed at the very end - separated by \\n.",
             "default": "M104 S0\nM140 S0\n;Retract the filament\nG92 E1\nG1 E-1 F300\nG28 X0 Y0\nM84"
         },
+        "material_bed_temp_wait": {
+            "default": true
```

Comment:
You left out the descriptions of these new machine settings, but perhaps they were added for all machine settings in between you first wrote this code and when the PR was merged.
Has been fixed.